opacus/validators/batch_norm.py

```diff
@@ -86,9 +93,10 @@ def _batchnorm_to_groupnorm(module: BATCHNORM) -> nn.GroupNorm:
         A default value of 32 is chosen for the number of groups based on the
         paper *Group Normalization* https://arxiv.org/abs/1803.08494
     """
-    return nn.GroupNorm(
-        math.gcd(32, module.num_features), module.num_features, affine=module.affine
-    )
+    if num_groups is None:
+        num_groups = math.gcd(32, module.num_features)
+
+    return nn.GroupNorm(num_groups, module.num_features, affine=module.affine)
```

Comment:
Might want to check that `num_features` is divisible by `num_groups`; and raise an exception otherwise.

Reply:
Actually I didn't do any validation on purpose - exception will be thrown from `nn.GroupNorm` constructor immediately and it will be a good one, so no reason to wrap it with our own

Reply:
Point taken!